codalab/lib/interactive_session.py

```diff
@@ -36,9 +36,13 @@ class InteractiveSession:
         "\n\n#\n"
         "# Choose the commands to use for cl run:\n"
         "#\n"
-        "# 1. Delete the lines of any unwanted commands.\n"
+        "# 1. Delete the lines of any unwanted commands. Removing all commands will not create a bundle.\n"
         "# 2. Add any additional commands on a new line above these instructions.\n"
         "#\n"
+        "# To quit without creating a bundle, run:\n"
```

Comment:
I'd get rid of this since it's vi specific and and is subsumed by the first comment above (removing all commands).